### ISSUE TITLE
Hyku: FileSets should have alt text metadata

### ISSUE THREAD

**Issue by ShanaLMoore** [samvera/hyku#2950]

**Comment by laritakr** [comment]
Refers to https://github.com/samvera/hyrax/issues/7236
